test-framework/common/src/main/java/io/quarkus/test/common/LauncherUtil.java

```diff
@@ -307,6 +312,31 @@ private void unableToDetermineData(String errorMessage) {
         }
     }
 
+    /**
+     * Used to drain the input of a launched process
+     */
+    private static class ProcessReader implements Runnable {
+
+        private final InputStream inputStream;
+
+        private ProcessReader(InputStream inputStream) {
+            this.inputStream = inputStream;
+        }
+
+        @Override
+        public void run() {
+            byte[] b = new byte[100];
+            int i;
+            try {
+                while ((i = inputStream.read(b)) > 0) {
+                    System.out.print(new String(b, 0, i, StandardCharsets.UTF_8));
```

Comment:
Do you think it is worth to make the printing configurable? `System.out.print vs System.err.print` depending on the input system in https://github.com/quarkusio/quarkus/blob/6acbce514fff84ff30792714d45e7e02e0b5422f/test-framework/common/src/main/java/io/quarkus/test/common/LauncherUtil.java#L60-L61?

Reply:
No one has ever asked for it and it's not been a problem so I prefer to leave it as is